test_cases/write_to_arbitrary_storage_location/arbitrary_location_write_simple.yaml

```diff
@@ -0,0 +1,7 @@
+description: "Simple variant of write to arbitrary storage location"
+issues:
+- id: "SWC-124"
+  count: 1
+  locations:
+  - bytecode_offsets: [294]
+    line_numbers: [26]
```

Comment:
Would 21 be another valid location to report this issue?

Reply:
I would say that's a separate overflow issue that I added to the file now.

Reply:
That makes sense